app/views/abouts/show.html.erb

```diff
@@ -0,0 +1,88 @@
+<div class="font-karla flex flex-col items-center w-full">
+    <div class="bg-blue-marine flex flex-col items-center px-5 pt-6 pb-10 w-full">
+        <div class="max-w-screen-sm text-center font-normal">
+            <p class="text-base mb-6 text-white">Brought to you by</p>
+            <%= inline_svg_tag "telos_labs_logo.svg", class: "mb-10 w-full" %>
+            <p class="text-white text-lg mb-6">We designed and developed this open-source app for the Rails community, you can contribute to the repo <a href="https://github.com/TelosLabs/rails-world" target="_blank" class="text-blue-teal">here</a>.</p>
+            <p class="text-white text-lg mb-10">Telos creates disruptive, purpose-driven software for visionary clients to enhance productivity and enrich lives.</p>
+            <div class="flex flex-row w-full items-center gap-5">
+                <a href="https://hi.teloslabs.co/" target="_blank" class="grow bg-blue-teal rounded-[10px] p-4 flex flex-row items-center justify-center">
```

Comment:
can we use [utm_params](https://en.wikipedia.org/wiki/UTM_parameters) for this link?

Reply:
@LuigiR0jas do you have access to google analytics? It would be nice to track visits from the app.

Reply:
Yes, we already have GA running on the Telos site. Good idea to add utm parameters